pkg/s3wrapper/client.go

```diff
@@ -23,12 +26,15 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3iface"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/cavaliercoder/go-cpio"
 	"github.com/go-openapi/swag"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
 const awsEndpointSuffix = ".amazonaws.com"
+const baseObjectName = "livecd.iso"
```

Comment:
I think that i already saw this defined somewhere? probably should be in one place?